mllib/src/main/scala/org/apache/spark/mllib/tree/configuration/BoostingStrategy.scala

```diff
@@ -88,4 +88,25 @@ object BoostingStrategy {
         throw new IllegalArgumentException(s"$algo is not supported by boosting.")
     }
   }
+
+  /**
+   * Returns default configuration for the boosting algorithm
+   * @param algo Learning goal.  Supported:
+   *             [[org.apache.spark.mllib.tree.configuration.Algo.Classification]],
+   *             [[org.apache.spark.mllib.tree.configuration.Algo.Regression]]
+   * @return Configuration for boosting algorithm
+   */
+  def defaultParams(algo: Algo): BoostingStrategy = {
```

Comment:
So you mean that remove the defaultParams(algoStr: String) and with the defaultParams(algo: Algo) instead?